maps-compose/src/main/java/com/google/maps/android/compose/MapUpdater.kt

```diff
@@ -152,15 +155,7 @@ internal inline fun MapUpdater(
         set(mapProperties.maxZoomPreference) { map.setMaxZoomPreference(it) }
         set(mapProperties.minZoomPreference) { map.setMinZoomPreference(it) }
         set(contentPadding) {
```

Comment:
This needs to change from `set()` to `update()`. Probably should move up to the other `update()` calls purely for consistency.

```suggestion
        update(contentPadding) {
```